helm/prometheus-rules/templates/platform/atlas/alerting-rules/promtail.rules.yml

```diff
@@ -41,7 +41,13 @@ spec:
             description: This alert checks if that the amount of failed requests is below 10% for promtail
             runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/promtail/
           expr: |
-            100 * (sum(rate(promtail_request_duration_seconds_count{status_code!~"2.."}[5m])) by (cluster_id, installation, provider, pipeline, namespace, job, route, instance) / sum(rate(promtail_request_duration_seconds_count[5m])) by (cluster_id, installation, provider, pipeline, namespace, job, route, instance)) > 10
+            100 * (
+                sum(rate(promtail_request_duration_seconds_count{status_code!~"2.."}[5m])) by (cluster_id, installation, provider, pipeline, namespace, job, route, instance)
+                /
+                sum(rate(promtail_request_duration_seconds_count[5m])) by (cluster_id, installation, provider, pipeline, namespace, job, route, instance)
+            ) > 10
+            # ignore this alert when alloy-logs is installed
+            unless on (cluster_id) count(up{service="alloy-logs"}) by (cluster_id) > 0
```

Comment:
Could you add a comment explaining why?

Reply:
Done.
I will add the "alloy-logs and promtail are both installed" alert in a follow-up PR.

Reply:
Nice